lib/graphql/input_object_type.rb

```diff
@@ -75,16 +75,14 @@ def coerce_non_null_input(value)
 
       arguments.each do |input_key, input_field_defn|
         field_value = value[input_key]
-        field_value = input_field_defn.type.coerce_input(field_value)
 
-        # Try getting the default value
-        if field_value.nil?
```

Comment:
@rmosolgo or @theorygeek I would love your :eyes: on this particular line.

Was the objective to use the default value if `value[input_key]` was undefined or was the goal to also catch the case where `coerce_input` returns `nil`?

I rewrote this code differently and tests still pass so it makes me wonder if we were missing a test or if the intended behaviour was to only use default value when argument was not provided a value.